src/transformers/onnx/features.py

```diff
@@ -173,6 +174,17 @@ class FeaturesManager:
             "question-answering",
             onnx_config_cls=BigBirdOnnxConfig,
         ),
+        "bigbird-pegasus": supported_features_mapping(
```

Comment:
After rebasing, please put this in alphabetical order, i.e. just below `bigbird`

Reply:
Rebased with main branch and modified this according to alphabetical order.

Reply:
Thank you! It seems there are still a few merge conflicts to resolve though and the PR has picked up many changes from the rebase. Would you mind resolving them please?

Reply:
Okay I will resolve these merge conflicts.